server/test/api/subsidy.js

```diff
@@ -0,0 +1,112 @@
+/*global describe, it, beforeEach, process*/
+
+var chai = require('chai');
+var chaiHttp = require('chai-http');
+var expect = chai.expect;
+chai.use(chaiHttp);
+
+var helpers = require('./helpers');
+helpers.configure(chai);
+
+var url = 'https://localhost:8080';
+var user = { username : 'superuser', password : 'superuser', project: 1};
+
+describe('The subsidy API, PATH : /subsidies', function () {
+  var agent = chai.request.agent(helpers.baseUrl);
+
+  var newSubsidy = {
+    account_id : 3626,
+    label : 'tested subsidy',
+    description : 'This is for the test',
+    value : 10  
+  };
+ 
+  var wrongSubsidy = {
+    account_id : 3626,
+    label : 'tested wrong subsidy',
+    description : 'This is for the wrong value test',
+    value : -10  
+  };
+
+  beforeEach(helpers.login(agent));
+
```

Comment:
For common issue like this, a separate pull request will be good, because almost all our test is dealing like this, so a separate pull request will walk through all those file and correct them if necessary.